rudder-webapp/SOURCES/rudder-webapp-apache

```diff
@@ -0,0 +1,5 @@
+# Apache modules
```

Comment:
Moved from rudder repo to keep all apache module for rudder-webapp in the same repo.